src_c/imageext.c

```diff
@@ -214,8 +214,9 @@ image_load_ext(PyObject *self, PyObject *arg)
         PyMem_Free(ext);
     }
 
-    if (surf == NULL) {
-        return RAISE(pgExc_SDLError, IMG_GetError());
+    if (surf == NULL){
+        SDL_ClearError();
+        return RAISE(PyExc_FileNotFoundError, "No such file or directory.");
```

Comment:
Maybe something like this would compile (untested):
```c
SDL_ClearError();
PyErr_SetString(PyExc_FileNotFoundError, "No such file or directory.");
return NULL;
```